### PR TITLE
PortfolioPage: Changed 'seconds' to 'second'.

### PR DESCRIPTION
**Pull request by katelandcastelin**

### CODE REVIEW

File: src/pages/PortfolioPage.tsx [hunk 1]
```diff
@@ -361,7 +361,7 @@ const PortfolioPage = (props: PropsFromRedux) => {
       </div>
       <div className={classes.sectionSpacer}>
         <Typography className={"align-center"} style={{fontWeight: 'bold'}} variant="h6">
-          updated {secondsAgo + ' seconds ago'}
+          updated {secondsAgo === 1 ? secondsAgo + ' second ago' : secondsAgo + ' seconds ago'}
```
Comment (JayWelsh) [inline src/pages/PortfolioPage.tsx]: Try to follow the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle wherever possible

In this case, let's ask ourselves what it is that we want to modify in the original string:

```
updated {secondsAgo + ' seconds ago'}
```

If we take this, we can see that we only really need to be able to switch a single word `second` <-> `seconds`

Technically, we could take this even further and say we actually only need to conditionally render an "s", but for the sake of readability let's just stick to wanting to switch between two single words ("second" and "seconds", instead of adding or leaving out a single letter "s").

So, basically, our conditional (in this case a ternary condition) only needs to impact whether we are rendering "seconds" or "second", we can just have `secondsAgo` being rendered once, instead of putting `secondsAgo` into the result of each of our conditions.

**Review by JayWelsh — APPROVED** [review]
